pro_tes/ga4gh/tes/models.py

```diff
@@ -708,8 +716,8 @@ class DbDocument(CustomBaseModel):
         tes_endpoint: External TES endpoint.
     """
 
-    task: TesTask = TesTask()
-    task_original: TesTask = TesTask(executors=[])
+    task: TesTask = TesTask()   # type: ignore
```

Comment:
eventhough `TesTask` has default value for all of its fields,
```python
...

class TesTask(CustomBaseModel):
    id: Optional[str] = Field(
        None,
        description="Task identifier assigned by the server.",
        examples=["job-0012345"],
    )
    state: Optional[TesState] = None
    name: Optional[str] = Field(None, description="User-provided task name.")
    description: Optional[str] = Field(
        None,
        description=(
            "Optional user-provided description of task for           "
            " documentation purposes."
        ),
    )
...
```
For some reason `mypy` didn't allow to instantiate as so. Providing value for each of its fields like `id = None` would defeat the purpose, and not inherit the default value.

Reply:
I thought `None` would implicitly be considered default, ig thats not the case, either way adding default fixes it.